templates/blog/blog_detail.html

```diff
@@ -47,5 +47,25 @@ <h1 class="title margin-top">{{ post.title }}</h1>
     </div>
   </article>
 
+</main>
+
+<main>
+
+  <h2 class="main-headline text-center">comments</h2>
+  <div>
+    {% for comment in comments %}
+       <p> <h5>{{comment.author}} commented:</h5>{{comment.message}}</p>
+    {% endfor %}
+  </div>
+  <div class="small-container margin-top flex">
+
+    <form action="" method="POST">
+      {% csrf_token %}
+      {{form}}
+      <button type="submit" value="Submit">Submit</button>
```

Comment:
a `<br>` would push the button to the right place.
example: `<br><button type="submit" value="Submit">Submit</button>`